boards/shields/zest_display_lcd/zest_display_lcd.overlay

```diff
@@ -40,3 +41,23 @@
 		};
 	};
 };
+
+&timers1 {
+	status = "okay";
+	st,prescaler = <1000>;
+};
+
```

Comment:
I think we should remove these lines. Prescaler support has been added to the Zest_Core

Reply:
It will be done in #5.